constraints.txt

```diff
@@ -113,71 +114,72 @@ jsonschema2md==1.1.0
 jsonstreams==0.6.0
 jupyter==1.0.0
 jupyter-console==6.6.3
-jupyter-events==0.9.0
-jupyter-lsp==2.2.3
-jupyter_client==8.6.0
-jupyter_core==5.7.1
+jupyter-events==0.10.0
+jupyter-lsp==2.2.4
+jupyter_client==8.6.1
+jupyter_core==5.7.2
 jupyter_server==2.13.0
-jupyter_server_terminals==0.5.2
-jupyterlab==4.1.2
+jupyter_server_terminals==0.5.3
+jupyterlab==4.1.5
 jupyterlab_pygments==0.3.0
-jupyterlab_server==2.25.3
+jupyterlab_server==2.25.4
 jupyterlab_widgets==3.0.10
 keyring==24.3.1
 kgcl-rdflib==0.5.0
-kgcl_schema==0.6.4
-kgx==2.3.0
+kgcl_schema==0.6.6
+kgx==1.7.2
```

Comment:
@hrshdhgd seems the latest ODK environment drops the KGX version again.. Any idea why?

Reply:
No idea ....will dig in some time this week. I don't recall which package was responsible last time.

Reply:
Would you be able to try an install https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full and figure out why kgx version dropped so hard? I am a bit worried about dropping the KGX version that much tbh..